meta.yaml

```diff
@@ -27,7 +27,8 @@ requirements:
 test:
   imports:
     - catalogbuilder
-    - catalogbuilder.scripts
     - catalogbuilder.cats
+    - catalogbuilder.scripts
```

Comment:
@Ciheim will be curious to see if fre-cli can still take the --config option because I had not moved configs during this restructuring. We can move it if fre-cli requires it, though it shouldn't be required. Please open a separate issue if fre-cli needs catalogbuilder.config